news_sentiment_analyser/extract_s3.py

```diff
@@ -0,0 +1,52 @@
+"""Extracts the csv from s3 bucket and returns as dataframe."""
+from io import BytesIO
+from os import environ as ENV
+
+from boto3 import client
+import pandas as pd
+from dotenv import load_dotenv
+
+
+def get_object_name(s3_client, bucket_name: str) -> str:
+    """Returns a list of object names for a specific bucket, at a specific time."""
+
+    objects = s3_client.list_objects(Bucket=bucket_name)
+
+    return [o["Key"] for o in objects["Contents"]][0]
```

Comment:
maybe some sort of error handling here, this should trigger when a file is uploaded, and files are deleted afterwords so should only have one, but just incase